db/index.js

```diff
@@ -40,7 +40,13 @@ const seedDB = () => {
     let data = new Album({
       id: i,
       artist: faker.name.jobDescriptor(),
-      album: [],
+      album:
```

Comment:
Instead of running the file each time to seed, you could also run it once and save the data to a file and then import that file in your package.json scripts. This works though.